converttodzi.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
```

Comment:
Does this file create a .dzi but searching every file/folder in `image/scollection-item` ?
Or does it actually cut up the images too?